Software/src/battery/JAGUAR-IPACE-BATTERY.cpp

```diff
@@ -42,11 +42,11 @@ static bool HVILBattIsolationError = false;
 static bool HVIsolationTestStatus = false;
 
 /* TODO: Actually use a proper keepalive message */
-CAN_frame ipace_keep_alive = {.FD = false,
+CAN_frame Simulated GWM NM = {.FD = false,
```

Comment:
Change this to:

```suggestion
CAN_frame simulated_GWM_NM = {.FD = false,
```

And also change the use of ipace_keep_alive in later lines.

You can also remove the Todo above.